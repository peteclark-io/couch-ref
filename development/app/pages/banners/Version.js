'use strict';

import React from 'react';
import { connect } from 'react-redux';
import {ThreeBounce} from 'better-react-spinkit';

import version from '../../../version.json'
import styles from './Splash.css';

const Version = React.createClass({

   render: function() {
     return (
      <div className={styles.splash}>
         <h1 className={styles.brand}>CouchRef</h1>
         <h3 className={styles.version}>{version.version}</h3>
      </div>
    );
  }
});

export default Version;
