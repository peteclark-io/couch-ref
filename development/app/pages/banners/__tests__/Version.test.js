'use strict';

import React from 'react';
import Version from '../Version.js';
import renderer from 'react-test-renderer';

test('Version matches the version.json file.', () => {
  const component = renderer.create(
    <Version />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
