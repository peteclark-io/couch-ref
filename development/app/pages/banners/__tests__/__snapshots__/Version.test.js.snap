exports[`test Version matches the version.json file. 1`] = `
<div
  className="splash">
  <h1
    className="brand">
    CouchRef
  </h1>
  <h3
    className="version">
    v0.0.0
  </h3>
</div>
`;
